### ISSUE TITLE
Login Fails sometimes

### ISSUE THREAD

**Issue by GoogleCodeExporter** [LeoLiXX/dataloadercliq#62]
```
The Extract process is scheduled to log in to Salesforce every 30 minutes and 
extract a file. Sometimes it's unable to login and gives out a time out error.
Error file is attached
```

Original issue reported on code.google.com by `pvir...@xede.com` on 22 Aug 2012 at 2:38

Attachments:
- [SF_DL_log-08-22-2012_@_10h15m04s.txt](https://storage.googleapis.com/google-code-attachments/dataloadercliq/issue-62/comment-0/SF_DL_log-08-22-2012_@_10h15m04s.txt)

**Comment by GoogleCodeExporter** [comment]
```
Do you have a proxy?  This kind of error looks like a network problem.
```

Original comment by `vijay.swamidass@gmail.com` on 1 Nov 2012 at 6:50

**Comment by GoogleCodeExporter** [comment]
Original comment by `vijay.swamidass@gmail.com` on 18 Nov 2012 at 1:01
- Changed state: **NoRepro**

**Comment by GoogleCodeExporter** [comment]
```
The issue was resolved with Data Loader 25.0 and CLIq 2.2.0
```

Original comment by `pvir...@xede.com` on 26 Nov 2012 at 8:54

**Comment by GoogleCodeExporter** [comment]
Original comment by `vijay.swamidass@gmail.com` on 28 Nov 2012 at 12:49
- Changed state: **Fixed**
